src/SprykerEco/Zed/Heidelpay/Persistence/Propel/Schema/spy_heidelpay.schema.xml

```diff
@@ -93,4 +93,37 @@
         </index>
     </table>
 
+    <table name="spy_payment_heidelpay_notification">
+        <column name="id_payment_heidelpay_notification" required="true" type="INTEGER" primaryKey="true" autoIncrement="true"/>
+
+        <column name="transaction_source" required="false" type="VARCHAR" />
+        <column name="transaction_channel" required="true" type="VARCHAR" />
+        <column name="transaction_response_type" required="false" type="VARCHAR" />
+        <column name="transaction_mode" required="false" type="VARCHAR" />
+
+        <column name="transaction_id" required="true" type="VARCHAR" />
+        <column name="unique_id" required="true" type="VARCHAR" />
+        <column name="short_id" required="false" type="VARCHAR" />
+        <column name="identification_source" required="false" type="VARCHAR" />
+
+        <column name="result_code" required="true" type="VARCHAR" />
+        <column name="result_timestamp" required="false" type="VARCHAR" />
+        <column name="result" required="true" type="VARCHAR" />
+        <column name="result_status" required="false" type="VARCHAR" />
+        <column name="result_reason" required="false" type="VARCHAR" />
+        <column name="result_return" required="false" type="VARCHAR" />
+
+        <column name="payment_code" required="false" type="VARCHAR" />
+        <column name="amount" required="false" type="VARCHAR" />
```

Comment:
so even amount is varchar?

Reply:
We are receiving this in xml and it converts to string by default. We can cast it to float or int but IMO it's not necessary. We won't use it as a value for any calculations or something.